networkx/algorithms/lowest_common_ancestors.py

```diff
@@ -35,6 +35,22 @@ def tree_all_pairs_lowest_common_ancestor(G, root=None, pairs=None):
     lcas : generator of tuples `((u, v), lca)` where `u` and `v` are nodes
         in `pairs` and `lca` is their lowest common ancestor.
 
+    Examples
+    --------
+
+    >>> G = nx.DiGraph()
+    >>> G.add_edges_from([(1, 3), (2, 4), (1, 2)])
```

Comment:
Just a nit, but an easy way to save a line!

```suggestion
    >>> G = nx.DiGraph([(1, 3), (2, 4), (1, 2)])
```